files/package.json

```diff
@@ -5,24 +5,20 @@
   "license": "MIT",
   "author": "",
   "scripts": {
-    "prepare": "yarn workspace <%= addonName %> run prepare",
-    "start": "npm-run-all --parallel start:*",
+    "prepare": "yarn build",
+    "build": "yarn workspace <%= addonName %> run build",
+
+    "start": "concurrently 'npm:start:*' --restart-after 5000 --prefix-colors cyan,white,yellow",
     "start:addon": "yarn workspace <%= addonName %> run start",
-    "start:test-app": "yarn workspace test-app run start",
-    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
-    "lint:addon": "yarn workspace <%= addonName %> run lint",
-    "lint:test-app": "yarn workspace test-app run lint",
-    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:fix:*",
-    "lint:fix:addon": "yarn workspace <%= addonName %> run lint:fix",
-    "lint:fix:test-app": "yarn workspace test-app run lint:fix",
-    "test": "npm-run-all --aggregate-output --continue-on-error --parallel \"test:!(watch)\"",
-    "test:watch": "npm-run-all --aggregate-output --continue-on-error --parallel test:watch:*",
```

Comment:
I removed the `watch` commands here as they're basically the same as `start`

Reply:
ember test --serve and ember serve are (nearly) the same, what's lost? 

> don't know if anybody would want to have this?

:shrug: as well